Gem/Code/Source/Performance/100KDraw10KEntityExampleComponent.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <Performance/HighInstanceExampleComponent.h>
+
+namespace AtomSampleViewer
+{
+    /*
+        This test loads a 46x46x46 lattice of entities with randomized meshes and materials. This test is used as a simple cpu performance stress test for Atom.
+
+        The assets that are applied to the entities are chosen from the 
+        "allow-list" of models and materials.
+    */
```

Comment:
Doesn't match the official commenting style: https://github.com/o3de/sig-core/blob/main/governance/API-Ref-Guidelines-Update.md#how-should-i-format-my-code-comments

Reply:
I'll update.